mongodbatlas/resource_mongodbatlas_alert_configuration_test.go

```diff
@@ -578,3 +603,19 @@ resource "mongodbatlas_alert_configuration" "test" {
 }
 	`, projectID, enabled, dataDogAPIKey, dataDogRegion)
 }
+
+func testAccMongoDBAtlasAlertConfigurationPagerDutyConfig(projectID, serviceKey string, enabled bool) string {
```

Comment:
Can we add another test for ops genie and victor ops? even if this is skipped for now, will be useful later in case of throubleshooting.

Reply:
Sure, thank you